pom.xml

```diff
@@ -2104,6 +2109,13 @@
                 </exclusion>
             </exclusions>
         </dependency>
+        <dependency>
+            <groupId>org.apache.httpcomponents.client5</groupId>
+            <artifactId>httpclient5</artifactId>
```

Comment:
This appears to be a duplicate of the older-style HTTP Client test dependency POM entry. Please remove it.

Reply:
removed